chatterbot/chatterbot.py

```diff
@@ -234,25 +234,24 @@ def learn_response(self, statement, previous_statement=None):
         """
         Learn that the statement provided is a valid response.
         """
+        from chatterbot.conversation import Statement
+
         if not previous_statement:
             previous_statement = statement.in_response_to
 
         if not previous_statement:
             previous_statement = self.get_latest_response(statement.conversation)
-            if previous_statement:
-                previous_statement = previous_statement.text
```

Comment:
I believe this line might still be needed. `self.get_latest_response` can return `None` which would cause `previous_statement.text` to error when accessed later on.

Reply:
This line has been shifted below.